public/js/draw/draw.js

```diff
@@ -389,3 +389,99 @@ function regionClicked(e) {
         prevY = position.y;
     }
 }
+
+
+/**
+ * Convert JSON data into SVG elements then populate the canvas with them.
+ */
+function jsonToSvg(jsonStr) {
```

Comment:
This function does two things, but the function name only represents one of them. This function name makes me think this just converts a JSON string to an SVG string, but this is not the case.